### ISSUE TITLE
Javascript in the Admin section for setting start and end dates

### ISSUE THREAD

**Issue by lucasjkr** [lucasjkr/multi-vote-polling-application#13]
In the admin panel, entering into the start date and end date field should bring up a date time picker. And when a selection is made, it should be stored in the database as such.

Currently the calendar icons are non-function. Dates are entered as YYYY-MM-DD, and 23:59:59 is appended to the end.

**Comment by Alofoxx** [comment]
give https://almsaeedstudio.com/themes/AdminLTE/index2.html a look

its a free MIT theme that you can use altogether or at least look at the forms page and see what libraries they use.

**Comment by lucasjkr** [comment]
Yeah, that looks overwhelming for this application... At least not unless it grows by leaps and bounds... Very reminiscent of opencart and wordpress though, I guess built their backends around this? Bookmarking for the future! 👍

**Comment by Alofoxx** [comment]
> Very reminiscent of opencart and wordpress though, I guess built their backends around this?

um not exactly.. this isnt a backend. Its just a HTML theme with some css and a few js libs for the charts and stuff. all the contend in the demo is just fake html stuff to show you what you /can/ do with the theme.

**Comment by lucasjkr** [comment]
Gotcha!

I'm basically learning my way through bootstrap at this point, so it'll probably be easier in the short term to just figure out how to implement their time/date pickers...
